src/less/filter-menu/base/filter-menu.less

```diff
@@ -0,0 +1,99 @@
+span.filter-menu {
+    display: inline-block;
+    max-width: 288px;
+    min-width: 144px;
+}
+
+.filter-menu__items {
+    .dropdown-base;
+
+    border-color: @filter-menu-border-color;
+    border-radius: 16px;
+    overflow: hidden;
+}
+
+.filter-menu__scroll-area {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height minus the footer
+    overflow-y: auto;
+    position: relative;
+}
+
+span.filter-menu__items[role="menu"] {
+    display: inline-block;
+}
+
+div.filter-menu__item[role^="menuitem"] {
```

Comment:
Also is there ever a menu item without role menuitem?

Reply:
Possibly. I'm not sure what you're alluding to in your question.

Reply:
Mostly to why we need the starts with menu item attribute. 
Assuming for specificity

Reply:
As Ian stated below, we have `menuitem`, `menuitemradio`, and `menuitemcheckbox` roles. All of them start with `menuitem`.

Reply:
For filters it will always be menuitemcheckbox (i.e. multi-select). I don't know if we ever have menuitemradio (i.e. single-select)...?